NOTES.md

```diff
@@ -1,3 +1,105 @@
+# Release 2.1.0-beta (Das Boot)
+
```

Comment:
We need to add some disclaimer about the fact that this is a beta release with known bugs and reliability issues. I'll write something.